enhancements/openshift-route-admission-policy.md

```diff
@@ -77,16 +75,35 @@ Addition of the new field requires no migration, as the default matches the curr
 
 ```go
 // RouteAdmissionPolicy is an admission policy for allowing new route claims.
-type RouteAdmissionPolicy string
+type RouteAdmissionPolicy struct {
+  // NamespaceOwnership describes how hostname claims across namespaces should
+  // be handled. The default is Strict.
+  NamespaceOwnership NamespaceOwnershipCheck
+}
+
+// NamespaceOwnershipCheck is a route admission policy component that describes
+// how hostname claims across namespaces should be handled.
+type NamespaceOwnershipCheck string
 
 const (
-    // AllowInterNamespaceClaimsRouteAdmissionPolicy allows routes to claim the same hostname across namespaces.
-    AllowInterNamespaceClaimsRouteAdmissionPolicy RouteAdmissionPolicy = "AllowInterNamespaceClaims"
+    // InterNamespaceAllowedOwnershipCheck allows routes to claim subdomains of the same hostname across namespaces.
+    InterNamespaceAllowedOwnershipCheck NamespaceOwnershipCheck = "InterNamespaceAllowed"
 
-    // StrictRouteAdmissionPolicy does not allow routes to claim the same hostname across namespaces.
-    // This is the default behavior 
-    StrictRouteAdmissionPolicy RouteAdmissionPolicy = "Strict"
+    // StrictNamespaceOwnershipCheck does not allow routes to claim the same hostname across namespaces.
+    StrictNamespaceOwnershipCheck NamespaceOwnershipCheck = "Strict"
 )
+
+// (Existing fields omitted, only newly proposed fields are visible)
+type IngressController struct {
+  // RouteAdmission defines a policy for handling new route claims (for example,
+  // to allow or deny claims across namespaces).
+  //
+  // The default policy is (in YAML):
+  //
+  //     namespaceOwnership: Strict
+  //
+  RouteAdmission *RouteAdmissionPolicy
```

Comment:
How specifically do we want to handle defaulting here (e.g. via admission control??

Will this be reflected on status?

cc @Miciah

Reply:
> How specifically do we want to handle defaulting here (e.g. via admission control??

Same as we handle defaulting for other endpoint publishing strategy, domain, etc.?  Adding an admission controller seems orthogonal.

> Will this be reflected on status?

Yeah, that would be the simplest approach, and consistent with what we have done as we have added other fields